scripts/priceBoundPSMDeploy.ts

```diff
@@ -0,0 +1,106 @@
+import hre, { ethers } from 'hardhat';
+import { expect } from 'chai';
+import config from './Config';
+/*
+
+Peg Stability Module
+
+Description: This module is used to manage the stability of the peg.
+
+Steps:
+  0 - Deploy FEI/VOLT PriceBoundPegStabilityModule
+*/
+
+const {
+  CORE,
+  FEI,
+  VOLT,
+  ORACLE_PASS_THROUGH_ADDRESS,
+  VOLT_FUSE_PCV_DEPOSIT,
+  /// fees
+  MINT_FEE_BASIS_POINTS,
+  REDEEM_FEE_BASIS_POINTS
+} = config;
+
+const feiReservesThreshold = ethers.utils.parseEther('10000000000');
```

Comment:
Does this not need multiplying by the number of decimals in Fei? When we set the reserves threshold for the ethPSM it was multiplied by 18 decimal places